.gitattributes

```diff
@@ -0,0 +1 @@
+*.go text eol=lf
```

Comment:
> This can (and IMO should) be turned off for all builds via git config --global core.autocrlf false prior to the checkout step.

How about using `.gitattributes`?
If we have it, we don't need to run `git config`.

Reply:
I suppose that works as well - in fact that looks even better! 😄